test/unit/VaryingCDFTest.cpp

```diff
@@ -0,0 +1,145 @@
+#include <catch.hpp>
+#include <iostream>
+
+#include <VaryingCDF.h>
+#include <Exceptions.h>
+#include <ContainerTools.hpp>
+#include <ParameterDict.h>
+#include <Function.h>
+#include <Gaussian.h>
+#include <math.h>
+
+
+using ContainerTools::ToString;
+using ContainerTools::GetValues;
+using ContainerTools::GetKeys;
+
+class Ploy : public Function{
```

Comment:
Again, maybe this could either be moved to src/function, or even better done with the ploy from the example?